src/xtt.h

```diff
@@ -24,6 +24,11 @@
 #include "channel.h"
 #include "packet.h"
 
+enum enftun_socket_type {
```

Comment:
`xtt.h` is for XTT specific stuff.  The enftun socket type is not XTT specific.  This should be elsewhere.

Reply:
Removed.